indico/modules/events/tracks/forms.py

```diff
@@ -28,8 +30,15 @@
 class TrackForm(IndicoForm):
     title = StringField(_('Title'), [DataRequired()])
     code = StringField(_('Code'))
+    default_session = QuerySelectField(_('Default session'), default='', allow_blank=True, get_label='title',
+                                       description=_('Indico will preselect this session whenever an abstract is '
+                                                     'accepted for the track'))
     description = IndicoMarkdownField(_('Description'), editor=True)
 
+    def __init__(self, event, *args, **kwargs):
```

Comment:
in all other forms requiring an event we pass it as a kwarg and pop it from `kwargs` before the super call so I'd do the same here